app/utils/rentry/wrapper.py

```diff
@@ -64,53 +73,95 @@ def __init__(self, text: str):
 
         try:
             response = self.new(text)
+            self.response = response
```

Comment:
I dont think we need this. The finally block should be able to use the local ```response``` variable directly.

Reply:
Ok removed